l10n_it_esigibilita_iva/migrations/14.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Sergio Zanchetta <https://github.com/primes2h>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.update_module_names(
+        env.cr,
+        [
+            ('l10n_it_esigibilita_iva', 'l10n_it_vat_payability'),
+        ]
+    )
+    openupgrade.rename_xmlids(
+        env.cr,
+        [
+            (
+                'l10n_it_vat_payability.view_tax_code_esigibilita_form',
```

Comment:
Qui mi pare ci sia un refuso: l'`xmlid` della vecchia vista è `l10n_it_esigibilita_iva.view_tax_code_esigibilita_form`.

Reply:
Hai ragione, mi è sfuggito.
Purtroppo non riesco a modificarlo perché è già stato fatto il merge, deve essere corretto nella PR principale.

Reply:
sì certo, ho commentato qui in modo da poterlo linkare nel commento della PR, vedi: https://github.com/OCA/l10n-italy/pull/1938#discussion_r561897779